go-controller/pkg/ovn/master.go

```diff
@@ -910,6 +891,37 @@ func (oc *Controller) syncNodesPeriodic() {
 	deleteChassis(oc.ovnSBClient, chassisMap)
 }
 
+func (oc *Controller) getJoinLRPAddresses(nodeName string) []*net.IPNet {
```

Comment:
should this function really be in this file?

Reply:
can you please suggest where it should go? It requires "oc" so it cannot be an utility function.

Reply:
maybe gateway.go? That's where the joinSwitch prefixes are. Also add a line of doc text explaining what the function does.

Reply:
sure. fixed